app/models/Item.scala

```diff
@@ -319,7 +319,7 @@ object Item {
 
   def deleteBefore(date: Date) {
     DB.withConnection { implicit c =>
-      val rows = SQL("select id from item where created < {created}").on('created -> date)
+      val rows = SQL("select id from item where created <= {created}").on('created -> date)
```

Comment:
Seems fine - although in practice there is unlikely to be much difference - the granularity of Date in the DB is milliseconds - so very few ever be equal